test/e2e/util/util.go

```diff
@@ -64,6 +64,26 @@ func GetTunedForNode(cs *framework.ClientSet, node *corev1.Node) (*corev1.Pod, e
 	return &podList.Items[0], nil
 }
 
+// GetTunedForNode returns a Pod that runs on a given node.
```

Comment:
Comment doesn't match what the function does.

Reply:
Fixed, sorry about that.